rpc/src/rpc.rs

```diff
@@ -115,6 +115,22 @@ type RpcCustomResult<T> = std::result::Result<T, RpcCustomError>;
 pub const MAX_REQUEST_BODY_SIZE: usize = 50 * (1 << 10); // 50kB
 pub const PERFORMANCE_SAMPLES_LIMIT: usize = 720;
 
+fn map_account_index(account_index: &AccountIndex) -> RpcAccountIndex {
```

Comment:
> I guess these can't be From trait impls without imposing otherwise unnecessary dependencies to either source crate?

Not actually sure. Lemme check, new to Rust, hand't even thought of that. Lemme ping you offline for context. 

Otherwise, yea can rename.

Reply:
i think that will be the case.  trait implementations must be in the same crate as either the trait or type are declared, in this case there are three crates to cover

Reply:
Renamed

Reply:
> I guess these can't be `From` trait impls without imposing otherwise unnecessary dependencies to either source crate?

Yes, that's right